### PR TITLE
CONSOLE-2445: deploymentConfig  i18n

### PR DESCRIPTION
**Pull request by zherman0**
[CONSOLE-2445](https://issues.redhat.com/browse/CONSOLE-2445): deploymentConfig  i18n

For completeness, this page needs PR7043, PR7056, and PR7042 to have all sections internationalized.

### CODE REVIEW

File: frontend/public/components/deployment-config.tsx [hunk 1]
```diff
@@ -103,70 +108,78 @@ export const menuActions: KebabAction[] = [
...
-          <DetailsItem label="Max Surge" obj={dc} path="spec.strategy.rollingParams.maxSurge">
-            {dc.spec.strategy.rollingUpdate.maxSurge || 1} greater than{' '}
-            {pluralize(dc.spec.replicas, 'pod')}
+          <DetailsItem
+            label={t('workload~Max surge')}
+            obj={dc}
+            path="spec.strategy.rollingParams.maxSurge"
+          >
+            {dc.spec.strategy.rollingUpdate.maxSurge || 1} {t('workload~greater than ')}
+            {t('workload~pod', { count: dc.spec.replicas })}
```
Comment (spadgett) [inline frontend/public/components/deployment-config.tsx:161]: This should be a single message with multiple parameters

File: frontend/public/components/deployment-config.tsx [hunk 2]
```diff
@@ -103,70 +106,82 @@ export const menuActions: KebabAction[] = [
...
+            label={t('workload~Max unavailable')}
             obj={dc}
             path="spec.strategy.rollingParams.maxUnavailable"
           >
-            {dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1} of{' '}
-            {pluralize(dc.spec.replicas, 'pod')}
+            {t('workload~{{maxUnavailable}} of pod', {
+              maxUnavailable: dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1,
+              count: dc.spec.replicas,
+            })}
```
Comment (spadgett) [inline frontend/public/components/deployment-config.tsx:153]: Does this display correctly? I think you need `{{count}}` in the message here?

```suggestion
            {t('workload~{{maxUnavailable}} of {{count}} pod', {
              maxUnavailable: dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1,
              count: dc.spec.replicas,
            })}
```

File: frontend/public/components/deployment-config.tsx [hunk 3]
```diff
@@ -103,70 +106,82 @@ export const menuActions: KebabAction[] = [
...
           </DetailsItem>
-          <DetailsItem label="Max Surge" obj={dc} path="spec.strategy.rollingParams.maxSurge">
-            {dc.spec.strategy.rollingUpdate.maxSurge ?? 1} greater than{' '}
-            {pluralize(dc.spec.replicas, 'pod')}
+          <DetailsItem
+            label={t('workload~Max surge')}
+            obj={dc}
+            path="spec.strategy.rollingParams.maxSurge"
+          >
+            {t('workload~{{maxSurge}} greater than pod', {
```
Comment (spadgett) [inline frontend/public/components/deployment-config.tsx:160]:
```suggestion
            {t('workload~{{maxSurge}} greater than {{count}} pod', {
```

Comment (zherman0) [reply frontend/public/components/deployment-config.tsx]: fixed

File: frontend/public/components/deployment-config.tsx (hunk 2 above)
Comment (zherman0) [reply frontend/public/components/deployment-config.tsx]: fixed - same problem in deployments

**Comment by zherman0** [comment]
@spadgett - fixed the maxSurge and maxUnavailable messages:
Screenshot_2020-11-06 example · Details · OKD

**Review by spadgett — APPROVED** [review]
/lgtm
/hold for approvals
/assign @yapei @ahardin-rh @sferich888

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**